docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  medmorph_backend:
+    image: medmorph_backend
+    container_name: medmorph_backend
+    restart: on-failure
+    ports:
+      - "3000:3000"
+    environment:
```

Comment:
`DATA_TRUST_SERVICE` should also be included 
https://github.com/mcode/medmorph-backend#configuration-requirements